gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/token/createAccount.ts

```diff
@@ -0,0 +1,56 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { ASSOCIATED_TOKEN_PROGRAM_ID, Token, TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST } from '../../../lib/contracts'
+
+export default class CreateAccount extends SolanaCommand {
+  static id = 'token:create_account'
+  static category = CONTRACT_LIST.TOKEN
+  static examples = ['yarn gauntlet token:create_account --network=devnet --address=<BASE_ADDRESS> <TOKEN>']
+
+  constructor(flags, args) {
+    super(flags, args)
+
+    this.requireFlag('address', 'Provide a token destination address')
+    this.require(!!args[0], 'Provide a token address')
+  }
+
+  execute = async () => {
+    const tokenAddress = new PublicKey(this.args[0])
+
+    const newAccountBase = new PublicKey(this.flags.address)
+    const associatedAcc = await Token.getAssociatedTokenAddress(
+      ASSOCIATED_TOKEN_PROGRAM_ID,
+      TOKEN_PROGRAM_ID,
+      tokenAddress,
+      newAccountBase,
+      true,
+    )
+
+    const ix = Token.createAssociatedTokenAccountInstruction(
+      ASSOCIATED_TOKEN_PROGRAM_ID,
+      TOKEN_PROGRAM_ID,
+      tokenAddress,
+      associatedAcc,
+      newAccountBase,
+      this.wallet.publicKey,
+    )
+
+    await prompt(`Continue to get or create new Token associated account to ${newAccountBase.toString()}`)
+    logger.loading('Creating account...')
+    const tx = await this.signAndSendRawTx([ix])
+
+    logger.success(`New account created at ${associatedAcc.toString()} on tx ${tx}`)
```

Comment:
Can we check if it exists before executing a tx?

If not quick to add just add a comment documenting further improvements as the next step.